app/services/auth/verify.rb

```diff
@@ -0,0 +1,13 @@
+module Auth
+  class Verify  < BaseService
```

Comment:
This class does "decode", not "verify" as its name. It doesn't call "add error" then how would you know if it fails?